myuw/dao/class_website.py

```diff
@@ -19,9 +20,15 @@ def _fetch_url(method, url):
         logger.error("_get_html_from_url(%s)==>%s" % (url, ex))
         return None
 
-    pool = get_con_pool("%s://%s" % (p.scheme, p.netloc), socket_timeout=2)
-    response = get_live_url(
-        pool, method, p.hostname, url, {'ACCEPT': 'text/html'})
+    headers = {'ACCEPT': 'text/html'}
+    if p.netloc == 'localhost':
```

Comment:
I think it's fairly clear, though there might be some resistance to it as the drive for "more realistic" mock data moves forward.  

I'd say the default would go to mock resources, to keep it in parity with the other live/mock resources.  I don't have a strong opinion about what urls are in mock data, but you might be able to avoid some questions if the links go to a working website.

If you want to keep the localhost solution, I'm not too worried about it as long as the security concerns are specifically defined and tested.